CONTRIBUTING.md

```diff
@@ -134,6 +134,27 @@ For Jekyll and its dependencies, because we already have the `Gemfile` in the re
 pushd site && bundle install && popd
 ```
 
+### Use  Visual Studio Code Remote - Containers for development
```

Comment:
Can you explain how `yarn link` with the editor would work in this case?

Reply:
I suppose I would need to do a "git clone" on the editor code.
And the just run "yarn link". In this case both "vega-lite" and "vega-editor" would be in the same Docker image.

But Docker is ephermal, so  I am not sure what happens to changed data outside "vega-lite" folder.
For the "vega-lite" folder VSCode manages the persistence.

So in scenarios of changing "both" simultaneously. ("vega-lite" and "vega-editor").

instead of "git clone", an additional host folder  (with vega-editor) could be mounted.
This could be as well configured in the devcontainer json file, but is user specific...

Reply:
not done for now